src/components/autocomplete/components/asyncAutocomplete/components/AsyncAutocompleteResultsCode.tsx

```diff
@@ -84,7 +84,7 @@ export default function AsyncAutocompleteResultsCode({
                 ? aooResult?.denominazioneAoo.toLocaleLowerCase()
                 : isUoCodeSelected && uoResult?.descrizioneUo
                 ? uoResult?.descrizioneUo.toLocaleLowerCase()
-                : visibleCode?.description
+                : (cfResult as any).businessName.toLocaleLowerCase()
```

Comment:
Why lowerCase? You should have already the businessName in lower case with first letter capitalized

Reply:
deleted lowercase on businessName